src/main/java/com/mattermost/integration/figma/input/file/notification/FileCommentNotificationRequest.java

```diff
@@ -0,0 +1,20 @@
+package com.mattermost.integration.figma.input.file.notification;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.*;
+
+@Getter
+@Setter
+@AllArgsConstructor
```

Comment:
pls take a look on lombok "@ Data" annotation  I believe it aggregates most of this annotations